GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -148,6 +149,47 @@ bool GPUTRDTracker_t<TRDTRK, PROP>::Init(TRD_GEOMETRY_CONST GPUTRDGeometry* geo)
     return false;
   }
 
+#ifdef GPUCA_ALIROOT_LIB
+  if (mBz < -90) {
+    GPUError("For AliRoot the magnetic field strength needs to be provided externally before initialization");
+    return false;
+  }
+#else
+  mBz = Param().BzkG;
+#endif
```

Comment:
Why cannot you read it from GPUParam in the AliRoot case?

Reply:
True, I think I can manually adjust the GPUSettingsEvent.solenoidBz and then also use the GPUParam in AliRoot. I did not see that before. Also the kG to T conversion was not correct :/